docs/dev/release.md

```diff
@@ -1,62 +1,29 @@
 # Making a release
 
-This is a guide on how to make a release of scanpy for maintainers.
+1. Go to GitHub’s [releases][] page
+2. Click “Draft a new release”
+3. Click “Choose a tag” and type the version of the tag you want to release, such as `1.9.6`
+4. Click “**+ Create new tag: 1.\<minor>.\<patch>** on publish”
+5. If the version is a pre-release version, such as `1.7.0rc1` or `1.10.0a1`, tick the “Set as a pre-release” checkbox
 
-## Checking that PyPI will accept it
+[releases]: https://github.com/scverse/scanpy/releases
 
-Sometimes PyPI will reject a build.
-You can locally check whether the build is acceptable with `twine check`.
-
-```shell
-# First, make a build
-python -m build
-
-# Now check that build
-twine check dist/*  # Assuming dist is otherwise empty
-```
-
-## Actually making release
-
-First, make sure you're working repository is clean and is on the commit you'd like to release from.
-Then follow these steps:
-
-```shell
-# Tag the commit with version info
-git tag {version}  # where version is a version number like "1.7.0"
-
-# Clear out old distributions
-rm -r dist
-
-# Build distributions and wheel
-python -m build
-# you can check that the previous step worked by installing from it's results
-# e.g. pip install dist/scanpy-{version}-py3-none-any.whl
-
-# Once you're confident the build looks good, push the tag to github
-git push upstream {version}
-
-# Upload wheel and code distribution to pypi
-twine upload dist/scanpy-{version}
-```
-
-### Checking the distributions
-
-If you're feeling cautious, you can:
+## After making a release
 
-- List the contents of a wheel with: `unzip -l dist/*.whl`
-- Make a release candidate. Tag these with `{version}rc1` (e.g. `1.7.0rc1`)
-- Upload them to <test.pypi.org> ([tutorial](https://packaging.python.org/en/latest/tutorials/packaging-projects/#uploading-the-distribution-archives))
+After *any* release has been made:
 
-## After making a release
+- Create a new release notes file for the next bugfix release.
+  This should be included in both dev and stable branches.
+- Create a milestone for the next release (in case you made a bugfix release) or releases (in case of a major/minor release).
+  For bugfix releases, this should have `on-merge: backport to 0.<minor>.x`,
+  so the [meeseeksdev][] bot will create a backport PR. See {doc}`versioning` for more info.
+- Clear out and close the milestone you just made a release for.
```

Comment:
It would be nice to automate this at some point

Reply:
For sure! Out of scope for this one?

Reply:
Yes